buildSrc/build.gradle

```diff
@@ -0,0 +1,26 @@
+plugins {
+    id 'groovy-gradle-plugin'
+}
+
+repositories {
+    maven {
+        url 'https://broadinstitute.jfrog.io/artifactory/plugins-snapshot'
+    }
+    gradlePluginPortal()
+}
+
+dependencies {
+    implementation 'com.diffplug.spotless:spotless-plugin-gradle:6.5.0'
```

Comment:
Can we use the same versions of plugins and dependencies found in Stairway and TCL
Reason - the versions used in those repos are updated and verified. Using disparate versions will lead to compatibility concerns later 

This comment applies to all build files & plugins